docs/rules/escape-user-inputs-in-sql-queries.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Escape User Inputs in SQL Queries
+ruleId: EscapeUserInputsInSQLQueries
+since: 3.17.0
+minJavaVersion: 1.1
+remediationCost: 10
```

Comment:
The remediation cost in the rule implementation is set to 5 minutes. This property needs to match with the remediation cost in the implementation.